website/src/pages/index.tsx

```diff
@@ -223,13 +221,15 @@ function Extensibility() {
             <FontAwesomeIcon size="3x" icon={faPlug} className="ml-2 mb-4 text-gray-800 dark:text-gray-200" />
           </div>
 
-          <p className="leading-relaxed">Container engines are plugged through extension points.</p>
-          <p className="leading-relaxed">JavaScript extensions can contribute new behaviour</p>
-          <p className="leading-relaxed">Reuse existing extensions directly in Podman Desktop</p>
+          <p className="leading-relaxed text-base">
+          Container engines are plugged through extension points.<br/>
+          JavaScript extensions can contribute new behaviour<br/>
+          Reuse existing extensions directly in Podman Desktop
+          </p>
 
           <div className="flex flex-col items-center">
             <div className="text-left my-4">
-              <p className="-ml-5">Current Podman Desktop plug-ins: Podman, Docker, Lima and CRC/OpenShift Local.</p>
+              <p className="-ml-5 text-base">Current Podman Desktop plug-ins: Podman, Docker, Lima and CRC/OpenShift Local.</p>
```

Comment:
maybe, instead of defining a new class, it should be the "default" color for dark theme which should be changed?

Reply:
I believe this is actually *missing* the class! Most <p> have `text-base` at the class.